exercises/all-your-base/all_your_base_test.rb

```diff
@@ -161,7 +165,7 @@ def test_leading_zeros
     digits = [0, 6, 0]
     input_base = 7
     output_base = 10
-    expected = [4, 2]
+    expected = WHAT TO DO?
```

Comment:
After looking at it with some fresh eyes, I think this is the only example that I'll have trouble implementing.

For the other examples I can check whether their input and bases are valid and generate a test on that. Also the empty / 0 input cases shouldn't be hard to catch either.

Because we ignore leading zeros in the input, the expected output for the given inputs is not included in the shared test data. I'm not sure how to handle this case in a way that would allow the input or bases to change without new generations breaking. Do you have any ideas on this one?

Thank you so much for your help and time!